projects/plugins/jetpack/modules/module-info.php

```diff
@@ -723,6 +723,25 @@ function jetpack_custom_jetpack_manage() {
 }
 add_action( 'jetpack_module_more_info_manage', 'jetpack_custom_jetpack_manage' );
 
+/**
+ * Post list info.
+ */
+function jetpack_post_list_link() {
+	echo esc_url( Redirect::get_url( 'jetpack-support-post-list' ) );
+}
+add_action( 'jetpack_learn_more_button_manage', 'jetpack_manage_more_link' );
+
+/**
+ * Post List description.
+ */
+function jetpack_post_list_info() {
+	esc_html_e(
+		'Enhance the classic view of the Admin section of your WordPress site.',
```

Comment:
I realize that's the package's main description, but I don't know if that's descriptive enough, since the package only modifies one page of the "admin section". 

How about:

```suggestion
		'Display extra information alongside each post in your dashboard's Posts screen.',
```

Reply:
Makes sense to me 👍